bigquery.js

```diff
@@ -18,18 +19,19 @@ var BigQuery = function() {
 BigQuery.prototype.insertInto = function(tableName, rows, options) {
   //Here I should get the table based on name
   // and insert in the right partition whenever the table is partitioned
-  rows.forEach((row) => {
+  this.table(tableName, (err, table) => {
     // First, we get the table object with all available information
-    this.table(tableName, (err, table) => {
-      if (err) {
-        console.log(`Sorry! We've got an error while fetching the table ${tableName}.\nERROR: ${err}`);
-        return;
-      }
-      var isPartitioned = BigQuery.isPartitioned(table);
-      var tablePartition = isPartitioned ? BigQuery.partitionName(row, table) : undefined;
+    if (err) {
+      console.log(`Sorry! We've got an error while fetching the table ${tableName}.\nERROR: ${err}`);
+      return;
+    }
+    var isPartitioned = BigQuery.isPartitioned(table);
+    rows.forEach((row) => {
+      let localOptions = extend({}, options);
```

Comment:
Nossa, nunca ia pensar nisso.

Reply:
`console.log(options)` is very often a good debugging tool, @helioalb

Reply:
Just to you know: javascript passing only reference of objects, not the value. Because that, this object needs to be cloned - the google lib change that.